weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterPanel.tsx

```diff
@@ -20,19 +21,17 @@ type FilterPanelProps = {
 };
 
 export const FilterPanel = (props: FilterPanelProps) => {
+  const {width: windowWidth} = useWindowSize();
+  // 400 = 60(sidebar) + 50 * 3(refresh, export, delete buttons) + 100 * 2 (compare + filter text)
+  let maxFilterBarWidth = windowWidth - 400;
```

Comment:
This number seems off, causing the display filters to get hidden even when there is more than enough room for them. Would be good to see if we can make it more accurate.